lib/beaker/shared/host_manager.rb

```diff
@@ -102,7 +102,7 @@ def run_block_on hosts = [], filter = nil, opts = {}, &block
         end
         if block_hosts.is_a? Array
           if block_hosts.length > 0
-            if opts[:run_in_parallel] == true
+            if run_in_parallel? opts, nil, nil
```

Comment:
Is there a reason we'd rather have users pass `nil` arguments instead of having `nil` default arguments?